subiquity/tests/api/test_api.py

```diff
@@ -1469,6 +1469,38 @@ async def test_can_create_unformatted_partition(self):
             v1resp = await inst.get('/storage')
             self.assertEqual([], match(v1resp['config'], type='format'))
 
+    @timeout()
+    async def test_guided_v2_resize_logical_middle_partition(self):
+        '''LP: #2015521 - a logical partition that wasn't the phyiscally last
```

Comment:
minor: physically

Reply:
I need to just leave the spellchecker on